Enhance navbar modified style.css #132

Kritikaagra opened the pull request.

tmo66 commented on static/home/css/style.css:
```diff
@@ -19,9 +19,18 @@ header{
+	text-align: center;
+	color: black;
+	border: none;
+	transition: 0.4s ease;
```
Don't change Global Variables.

tmo66 commented on static/home/css/style.css:
```diff
@@ -32,13 +41,19 @@ a{
+	font-size: 18px;
 }
-a:hover{
+.navbar a:hover{
```
Don't make changes here, do these changes in `navbar a: hover`.

tmo66 commented on static/home/css/style.css:
```diff
@@ -54,7 +69,7 @@ a:hover{
 	/* background: var(--theme-dark-color-0); */
 	/* background: #666; */
-  }
+  } 
```
Don't add extra whitespace and an extra line.

tmo66 commented on static/home/css/style.css:
```diff
@@ -101,17 +116,21 @@ nav{
 	z-index: 2;
+	font-family:'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;	
 }
+
```
Don't add an extra line here.

tmo66 commented on static/home/css/style.css:
```diff
@@ -101,17 +116,21 @@ nav{
 	cursor: pointer;
 	transition: 0.5s;
+	margin: 5.5px;
+	
```
Don't add an extra line here.

tmo66 commented on static/home/css/style.css:
```diff
@@ -101,17 +116,21 @@ nav{
 }
 .navbut:hover{
-	background-color: red;
+	background-color:red;
```
Don't remove the spacing between both. It's how the code is formatted throughout the project.

tmo66 commented on static/home/css/style.css:
```diff
@@ -182,7 +204,7 @@ nav{
 /*navbar in full size*/
 .navbar{
 	float: left;
-	/*margin: 5px 0;*/
```
Don't remove this.

tmo66 commented on static/home/css/style.css:
```diff
@@ -191,28 +213,25 @@ nav{
 }
 .navbar li{
 	display: inline;
-	/*padding: 0 5px;*/
```
Don't remove this.

tmo66 commented on static/home/css/style.css:
```diff
@@ -191,28 +213,25 @@ nav{
 	background-color: rgba(255,140,0,0.9);
-	box-sizing: border-box;
-	height:0px;
+	/*box-sizing: border-box;*/
```
Don't delete anything which is not created by you. Just put them in the comment. And delete all those which are written by you but later on commented.

tmo66 reviewed (changes requested):
Make all these necessary changes.

tmo66 commented on static/home/css/style.css:
```diff
@@ -136,21 +155,23 @@ nav{
 	position: relative;
-	padding: 3px 10px;
+	/*margin: 3px 5px;*/
+	padding: 13.5px 11px;
```
Not sure about 13.5 px...it's too much!

Kritikaagra commented:
done with all necessary changes...
set all global variable as their previous values
remove all extra lines and comments added by me
also added lines I deleted ... and make them as comment to make changes

If I decrease padding from 13.5, the red colour on hover does not cover the whole space and leave a line at bottom.

tmo66 commented on static/home/css/style.css:
```diff
@@ -136,21 +139,25 @@ nav{
 .navbar1 li{
 	display: inline;
-	padding: 0 5px;
+	padding: 0px 0px;
```
No need to write this..

tmo66 commented on static/home/css/style.css:
```diff
@@ -136,21 +139,25 @@ nav{
 .navbar1 .button:hover{
-	background-color: red;
 	color: white;
-	padding: 3px 12px;
```
Don't remove these two-line, just put them in comment.

tmo66 commented on static/home/css/style.css:
```diff
@@ -159,9 +166,10 @@ nav{
+	height: 100%;
 	margin: auto;
-	background: red;
+	background: white;
```
No need to make changes here, as this whole part is already commented out.

tmo66 commented on static/home/css/style.css:
```diff
@@ -234,6 +246,9 @@ nav{
 .navbar2 li a{
 	display: block;
 }
+.navbar2 a:hover{
```
Don't make these changes, for now, we are handling navbar2 in a different issue.

tmo66 reviewed (changes requested):
Just make these small changes, everything else looks good to me! And also after the commit squash all the commits in one.

tmo66 commented:
To squash your commits, follow these instructions:
* `git rebase -i HEAD~<no. of commits you made in total>`
* Except for first `pick`, replace all `pick` with `squash` in the file opened.
* Close the file after saving it.
* In the new file opened, remove all commit messages and write a new one and then close this file too after saving.
* Force push your changes. (`git push origin +<your-branch-name>`).

garg3133 commented:
Thank you @Kritikaagra for working on this. The enhancements were good but the navbar was not being displayed properly on small screens due to which I was reluctant to merge this PR. But then I thought, why not rebuild the entire navbar from scratch as it was very badly implemented (you must have seen there were two navbars, one for small screens and the other of larger screens). So, I spent some time on it and re-built it.

But, as your enhancements were good and I've taken some idea from this PR itself while rebuilding it, I'm still merging this PR so that you get your points for GSSoC. And I'm really sorry you didn't get your changes in the master branch. Hope you'll understand and continue making your amazing contributions. Thanks!

garg3133 commented:
Also, you should never make any changes directly into the master branch.

Kritikaagra commented:
Thank you @garg3133
And sorry for making changes in the master branch. I will take care this next time.
